lib/warrant/util.rb

```diff
@@ -21,6 +21,11 @@ def normalize_params(params)
                 case params
                 when Hash
                     params.each_with_object({}) do |(k, v), new_opts|
+                        if k.to_s == "context"
```

Comment:
Can you add a comment here (for our future selves) explaining why we're skipping this behavior for the `context` property?